rero_ils/modules/items/api/circulation.py

```diff
@@ -1288,6 +1312,7 @@ def is_requested_by_patron(self, patron_barcode):
 
     def is_loaned_to_patron(self, patron_barcode):
         """Check if the item is loaned by a given patron."""
+        # TODO: check if this method is still used and delete it if not.
```

Comment:
seems not used anywhere else. Code coverage test will prevent you ;-)

Reply:
removed